test/runner/transform.js

```diff
@@ -0,0 +1,45 @@
+const crypto = require('crypto');
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const pirates = require('pirates');
+const babel = require('@babel/core');
+const version = 0;
+const cacheDir = path.join(os.tmpdir(), 'playwright-transform-cache');
+
+/**
+ * @param {string} content 
+ * @param {string} filePath 
+ * @return {string}
+ */
+function calculateCachePath(content, filePath) {
+  const hash = crypto.createHash('md5').update(content).update(filePath).update(String(version)).digest('hex');
+  const fileName = path.basename(filePath, path.extname(filePath)).replace(/\W/g, '') + '_' + hash;
```

Comment:
just use hash?

Reply:
Copied this from jest. putting the name in the cache path makes it way easier to debug issues with the cache system.